tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -407,3 +419,37 @@ Get User Notebook PVC Name
     ${safe_username} =   Get Safe Username    ${username}
     ${notebook_pod_name} =   Set Variable  jupyterhub-nb-${safe_username}-pvc
     [Return]    ${notebook_pod_name}
+
+Clone Repo
+    [Documentation]    Clones git repo
+    [Arguments]    ${REPO_URL}
+    Navigate Home (Root folder) In JupyterLab Sidebar File Browser
+    Open With JupyterLab Menu    Git    Clone a Repository
+    Input Text    //div[.="Clone a repo"]/../div[contains(@class, "jp-Dialog-body")]//input    ${REPO_URL}
+    Click Element    xpath://div[.="CLONE"]
+
+
+Clone Repo and Return Error Message
+    [Arguments]    ${REPO_URL}
+    Clone Repo    ${REPO_URL}
+    Wait Until Page Contains    Cloning...    timeout=5s
+    ${err_msg} =    Did Error Occur While Cloning Git Repo
+    [RETURN]    ${err_msg}
+
+Get Directory Name From Git Repo URL
+    [Documentation]    Returns directory name from repo link
+    [Arguments]    ${REPO_URL}
```

Comment:
Arguments should not be in capital letters

Reply:
Done